src/app.controller.ts

```diff
@@ -28,7 +28,7 @@ export class AppController {
     };
   }
 
-  @UseGuards(BasicAuthGuard)
+  //@UseGuards(BasicAuthGuard)
```

Comment:
Is this how you avoid the certificate issue?

Reply:
By using http not https for my site I avoided the certificate issue.
When we remove this @UseGuards(BasicAuthGuard) then we don't need to send in header basic token from frontend.